src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -28,10 +38,30 @@ public void findAllBusinesses() {
     businesses.add(new Listing("The Shiny Bean Coffee & Tea", new MapLocation(40.496328, -79.944862), 4.9, "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", "https://theshinybean.com/"));
     businesses.add(new Listing("Weisshouse", new MapLocation(40.456684, -79.925499), 4.3, "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", "https://www.weisshouse.com/"));
   }
+  
+  // Function is only here until the places API is implemented,
+  // and for test file to be able to set any static business LinkedList we want
+  // to try to use
+  public void setAllBusinesses(List<Listing> allBusinesses){
+   businesses = allBusinesses;
+  }
 
   public void eliminateBigBusinesses() {
-    // TODO: Parse big business list and remove big businesses from businessList
-
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("BigBusinesses").addSort("business", SortDirection.DESCENDING);
```

Comment:
What is this sorting by? Is it alphabetical/reverse-alphabetical?

Reply:
I do not know why I am sorting here as it does not help with anything so I will remove it